nested-spire/README.md

```diff
@@ -0,0 +1,210 @@
+       -parentID "spiffe://example.org/spire/agent/x509pop/$(fingerprint root/agent/agent.crt.pem)" \
+       -spiffeID "spiffe://example.org/nestedA" \
+       -selector "docker:label:org.example.name:nestedA-server" \
+       -downstream \
+       -ttl 3600
+```
+
+The `-parentID` flag contains the SPIFFE ID of the `root-agent`. The SPIFFE ID of the `root-agent` is created by the [x509pop Node Attestor](https://github.com/spiffe/spire/blob/master/doc/plugin_server_nodeattestor_x509pop.md) plugin which defines the SPIFFE ID as `spiffe://<trust domain>/spire/agent/x509pop/<fingerprint>`. To make the entry clear to the reader, it is expressed as `fingerprint root/agent/agent.crt.pem` indicating that it contains the fingerprint of the agent certificate.
+The other point to highlight is the `-downstream` option. This option when set, indicates that the entry describes a downstream SPIRE server.
+
+## Run the Scenario
+
+Use the `set-env.sh` script to run all the services that compound the scenario. All the configurations described in the previous sections are illustrated in the different services.
+Ensure that the current working directory is `.../spire-tutorials/nested-spire` and run:
+
+```console
+    bash scripts/set-env.sh
+```
+
+Once the script is completed, in another terminal run the following command to review the logs from all the services
+
+```console
+    docker-compose logs -f -t
+```
+
+
+# Part 2: Test the Deployments
+
+Now that the SPIRE deployments are ready, let's test the scenario that we've configured.
+
+## Create Workload Registration Entries
+
+To test the scenario we create two workload registration entries, one entry per each of the nested SPIRE deployment. The goal of the test case is to demonstrate that SVIDs created in a nested configuration are valid in the entire trust domain, not only in the scope of the server that originated the SVID.
+
+```console
+   # Workload for nestedA deployment
+   docker-compose exec -T nestedA-server \
+       /opt/spire/bin/spire-server entry create \
+       -parentID "spiffe://example.org/spire/agent/x509pop/$(fingerprint nestedA/agent/agent.crt.pem)" \
+       -spiffeID "spiffe://example.org/nestedA/workload" \
+       -selector "unix:uid:1001" \
+       -ttl 0
+
+   # Workload for nestedB deployment
+   docker-compose exec -T nestedB-server \
+       /opt/spire/bin/spire-server entry create \
+       -parentID "spiffe://example.org/spire/agent/x509pop/$(fingerprint nestedB/agent/agent.crt.pem)" \
+       -spiffeID "spiffe://example.org/nestedB/workload" \
+       -selector "unix:uid:1001" \
+       -ttl 0
+```
+
+The examples uses the `$(fingerprint nestedA/agent/agent.crt.pem)` notation again to show that the `-parentID` flag is set to the SPIFFE ID of the nested SPIRE Agent. TTL equals to zero indicates that the default value of 3600 seconds will be used. Finally, in both cases there is an unix selector to identify any process that is run by the user with id 1001.
```

Comment:
Since UNIX is pronounced "yoo-nix" it's "a unix" selector. But if you call for a car using an app, it's "getting an Uber" (oo-brr).

Please check my changes for any wrong information.

```suggestion
The examples use the `$(fingerprint nestedA/agent/agent.crt.pem)` form again to show that the `-parentID` flag specifies the SPIFFE ID of the nested SPIRE Agent. The TTL flag value of zero indicates that the default value of 3600 seconds will be used. Finally, in both cases the unix selector assigns the SPIFFE ID to any process with a uid of 1001.
```